apps/test-providers/src/ui/ViewportContent.tsx

```diff
@@ -6,9 +6,12 @@ import * as React from "react";
 import { DefaultViewOverlay, UiFramework } from "@itwin/appui-react";
 import { IModelApp, ScreenViewport, ViewState } from "@itwin/core-frontend";
 import { ViewportComponent } from "@itwin/imodel-components-react";
+import { viewWithUnifiedSelection } from "@itwin/presentation-components";
 
 type ViewportComponentProps = React.ComponentProps<typeof ViewportComponent>;
 
+const UnifiedViewport = viewWithUnifiedSelection(ViewportComponent);
```

Comment:
I'd rather we use the `@itwin/unified-selection` package than the deprecated `viewWithUnifiedSelection`

Reply:
This is just for local debugging and testing. All presentation related APIs will be removed from appui test-app.